ui/admin/app/components/form/field/list-wrapper/index.hbs

```diff
@@ -0,0 +1,180 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Form::Fieldset
+  @layout={{@layout}}
+  @isRequired={{@isRequired}}
+  @isOptional={{@isOptional}}
+  as |F|
+>
+
+  {{yield F}}
+
+  {{#let (unique-id) (unique-id) as |keyHeaderID valueHeaderID|}}
```

Comment:
Asking for learning purposes: when we do, (unique-id) (unique-id) as |keyHeaderID valueHeaderID|, it's because we are generating two new bindings (keyHeader & valueHeader) with a unique ID values, is that right?

Reply:
yup that is correct